### PR TITLE
DEVDOCS-3240: Add tested version reference to indicate breaking change at firebase9

### PR DESCRIPTION
**Pull request by slsriehl**
# [DEVDOCS-3240](https://jira.bigcommerce.com/browse/DEVDOCS-3240)

## What changed?
* revised nextjs tutorial to npm i firebase@^8xx, because v9 breaks our app.

### CODE REVIEW

**Review by markcmurphy — APPROVED** [review]

**Review by bc-chaz — CHANGES_REQUESTED** [review]
@slsriehl rather than adding in an explicit firebase package, please just add a note saying something like doing `npm install x package` will install the latest major version which may break. If so, please refer to the package.json and install that version.

The problem with your current approach is that we will constantly need to update the docs. We currently have a ticket in this sprint to update firebase to v9 so this will break in short order. Please see our discussion yesterday in the slack channel.

File: docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md [hunk 1]
```diff
@@ -30,16 +30,16 @@ This step demonstrates how to integrate the sample app with [Cloud Firestore](ht
 
 ## Install npm packages
 
-If using Firebase, install `firebase`, `jsonwebtoken`, and `swr` npm packages.
+If using Firebase, install `firebase@^8.2.6`, `jsonwebtoken`, and `swr` npm packages.
```
Comment (bc-chaz) [inline docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md:33]: please don't merge this

**Review by markcmurphy — COMMENTED** [review]
Changing approval based on @bc-chaz feedback

File: docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md [hunk 2]
```diff
@@ -75,6 +75,18 @@ npm install --save-dev babel-plugin-styled-components @types/node @types/react t
 
 [@types/node](https://www.npmjs.com/package/@types/node) and [@types/react](https://www.npmjs.com/package/@types/react) contain TypeScript type definitions for Node.js and React.js respectively.
 
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme: info -->
+
+> You can view a list of all the tested package versions in the [package.json file on the Step 1 branch](https://github.com/bigcommerce/sample-app-nodejs/blob/step-1-app-foundation/package.json) of this sample app's repo.
```
Comment (bc-chaz) [inline docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md:84]: 👍 thanks for linking to the package file.

qq: what does `theme: info` look like here?

**Review by bc-chaz — APPROVED** [review]
one minor question, otherwise LGTM 👍

File: docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md (hunk 2 above)
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md:84]: same as the stencil docs ref you linked -- blue. https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil#installing-on-linux

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**